src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -20,6 +20,38 @@ public class HelpWindow extends UiPart<Stage> {
     public static final String USERGUIDE_URL = "https://ay2223s2-cs2103-w17-1.github.io/tp/UserGuide.html";
     public static final String HELP_MESSAGE = "Refer to the user guide:\n" + USERGUIDE_URL;
 
+    public static final String USER_GUIDE = "Greeting! There is a quick and easy guide for you to use the MATHUTORING. "
+            + "Below are the functions supported.\n"
+            + "Commands with parameter(s):\n"
+            + "1. add: Adds a student to the student list. \n"
+            + "2. addScore: Adds a score for a selected student.\n"
+            + "3. deleteScore: Deletes a score for a selected student.\n"
+            + "4. addTask: Adds a task for a selected student.\n"
+            + "5. deleteTask: Deletes a task for a selected student.\n"
+            + "6. check: Checks the task(s) and score(s) for a selected student.\n"
+            + "7. delete: Deletes a student from the student list.\n"
+            + "8. edit: Edits the student’s information.\n"
+            + "9. find: Finds student(s) from the student list by their name(s).\n"
+            + "10. filter: Filters out the matching result based on tag(s).\n"
+            + "11. markComplete: Marks task as complete.\n"
+            + "12. markInProgress: Marks task as in progress.\n"
+            + "13. markLate: Marks task as late.\n"
+            + "14. exportP: Exports a student's progress as PDF file.\n"
```

Comment:
The commands can change from camelCase to lowercase

Reply:
Okie, but may I ask if our UG also need to change to lowercase to keep consistent?